### ISSUE TITLE
Unable to pod install when including 1PasswordExtension due to type failure

### ISSUE THREAD

**Issue by brianmichel** [CocoaPods/CocoaPods#5270]
### Command

```
/Users/brianmichel/.rbenv/versions/2.2.0/bin/pod install
```

### Report

* What did you do?
I ran `bundle exec pod install` to ensure that I was using the rc version of the CocoaPods gem.

* What did you expect to happen?
I expected `bundle exec pod install` to correctly install all of my dependencies.

* What happened instead?
I believe CocoaPods installed the dependencies but broke down when trying to integrate it into the workspace, producing the following stack.

### Stack

```
   CocoaPods : 1.0.0.rc.2
        Ruby : ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin15]
    RubyGems : 2.4.5
        Host : Mac OS X 10.11.4 (15E65)
       Xcode : 0.1 (1)
         Git : git version 2.3.1
Ruby lib dir : /Users/brianmichel/.rbenv/versions/2.2.0/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 6594d96acdc4549dfbe16317131c2b1699f07c12
```

### Plugins

```
cocoapods-deintegrate : 1.0.0.rc.2
cocoapods-plugins     : 1.0.0.rc.1
cocoapods-search      : 1.0.0.rc.1
cocoapods-stats       : 1.0.0.rc.1
cocoapods-trunk       : 1.0.0.rc.1
cocoapods-try         : 1.0.0.rc.1
```

### Podfile

```ruby
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '8.0'
use_frameworks!

target 'Broken1PasswordExtensionInstallation' do
  pod '1PasswordExtension', '~> 1.8'
end
```

### Error

```
TypeError - Unable to convert Ruby value `"1PasswordExtension"' into a CFTypeRef.
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:383:in `RubyValueToCFTypeRef'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:402:in `block in RubyHashToCFDictionary'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:400:in `each'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:400:in `RubyHashToCFDictionary'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:372:in `RubyValueToCFTypeRef'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:402:in `block in RubyHashToCFDictionary'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:400:in `each'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:400:in `RubyHashToCFDictionary'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:372:in `RubyValueToCFTypeRef'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:402:in `block in RubyHashToCFDictionary'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:400:in `each'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:400:in `RubyHashToCFDictionary'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi/core_foundation.rb:36:in `RubyHashPropertyListWrite'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist/ffi.rb:42:in `write_to_path'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/plist.rb:46:in `write_to_path'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-1.0.0.rc.2/lib/xcodeproj/project.rb:332:in `save'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/installer.rb:696:in `block in write_pod_project'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/user_interface.rb:144:in `message'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/installer.rb:688:in `write_pod_project'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/installer.rb:173:in `block in generate_pods_project'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/user_interface.rb:63:in `section'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/installer.rb:167:in `generate_pods_project'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/installer.rb:119:in `install!'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/command/install.rb:37:in `run'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/claide-1.0.0.rc.1/lib/claide/command.rb:334:in `run'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/lib/cocoapods/command.rb:50:in `run'
/Users/brianmichel/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-1.0.0.rc.2/bin/pod:55:in `<top (required)>'
/Users/brianmichel/.rbenv/versions/2.2.0/bin/pod:23:in `load'
/Users/brianmichel/.rbenv/versions/2.2.0/bin/pod:23:in `<main>'
```

**Comment by brianmichel** [comment]
Sample project available here, https://github.com/brianmichel/CocoapodsBroken1PasswordExtension

**Comment by orta** [comment]
![screen shot 2016-05-08 at 4 00 34 pm](https://cloud.githubusercontent.com/assets/49038/15100087/06a832be-1536-11e6-9d08-bda143b3993f.png)

Interesting, I don't get this in the Mac App

**Comment by brianmichel** [comment]
Hmmmmm. I'm actually seeing this happen on multiple libraries intermittently, let me do some more digging until I find something a bit more concrete.

**Comment by brianmichel** [comment]
Certainly seeing some intermittent failures on my system both through bundle and not when using the version 1.0.0.rc.2

**Comment by brianmichel** [comment]
This are accompanied by `error: tool 'xcode-select' not found` messages however running `which code-select` returns `/usr/bin/xcode-select`

**Comment by orta** [comment]
Might be worth exploring these too: https://github.com/CocoaPods/CocoaPods/search?q=into+a+CFTypeRef&type=Issues&utf8=✓

**Comment by brianmichel** [comment]
Ah yeah, smells exactly the same, will give it a go, thanks @orta!

**Comment by segiddins** [comment]
This is an issue with that particularly ruby version (a bug in their FFI implementation). It's fixed either by updating to the latest 2.2.x or 2.3 version.
